### ISSUE TITLE
Can't install it in Ubuntu 20.04, either by PPA or manually

### ISSUE THREAD

**Issue by Musrar** [fossfreedom/coverart-browser#403]
Hello, I seem to be unable to install the plugins, be it manually or via PPA.
I have tried via PPA, previously deleting the folders that were created when I first tried it manually, and the terminal just gives me a "couldn't find these packages".
Then, manually, if I try to install it spams:
```
installing languages to /usr/share/locale/
installing ca
Installing the glib schema (password needed)
```
: (

**Comment by fossfreedom** [comment]
coverart no longer works with recent versions of python/gtk and ubuntu 20.04 due to elimination of webkit and other gtk changes.

It will need reworking to make it compatible again - something which I don't have the time to-do :(

**Comment by Musrar** [comment]
Gotcha, no worries! Thanks for answering :)

**Comment by Musrar** [comment]
Oupsie, must close this.

**Comment by ghost** [comment]
Vou falar em portugues mesmo. hehehe

eu consegui fazer funcionaaarrr.. ta MUITO SHOWWW!! vejam

![Captura de tela de 2020-08-18 00-26-47](https://user-images.githubusercontent.com/2234645/90467056-c0db4080-e102-11ea-9f83-777aebed2990.png)

eu consegui no PureOS (Debian 100% livre). o que eu fiz foi apenas instalar manualmente o pacote gir1.2-webkit-3.0 e suas dependencias.

eu baixei por aqui:  https://packages.debian.org/stretch/amd64/gir1.2-webkit-3.0/download

e instalei assim

sudo dpkg -i gir1.2-webkit-3.0_2.4.11-3_amd64.deb

tem as dependencias tbm q seguem a mesma parada.

**Comment by linuxitomio** [comment]
hola @wellton .
uso ubuntu 20.04 y no me deja descargarlo en software .

**Comment by linuxitomio** [comment]
they have unmet dependencies

**Comment by ghost** [comment]
> hola @wellton .
> uso ubuntu 20.04 y no me deja descargarlo en software .

Hello Linuxitomio, 

1) open terminal.

2) type: rhythmbox

3) send to here the errors in terminal.

**Comment by linuxitomio** [comment]
rhythmbox

(rhythmbox:14426): Gtk-WARNING **: 09:50:45.804: actionhelper: action app.play-repeat can't be activated due to parameter type mismatch (parameter type NULL, target type b)

(rhythmbox:14426): Gtk-WARNING **: 09:50:45.805: actionhelper: action app.play-shuffle can't be activated due to parameter type mismatch (parameter type NULL, target type b)

** (rhythmbox:14426): WARNING **: 09:50:54.905: Error importing plugin 'coverart_browser':
Traceback (most recent call last):
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/deloco/.local/share/rhythmbox/plugins/coverart_browser/coverart_browser.py", line 32, in <module>
    from coverart_browser_source import CoverArtBrowserSource
  File "/home/deloco/.local/share/rhythmbox/plugins/coverart_browser/coverart_browser_source.py", line 33, in <module>
    from coverart_album import AlbumManager
  File "/home/deloco/.local/share/rhythmbox/plugins/coverart_browser/coverart_album.py", line 1590
    async = rb.Loader()
          ^
SyntaxError: invalid syntax

(rhythmbox:14426): libpeas-WARNING **: 09:50:54.906: Error loading plugin 'coverart_browser'
/usr/lib/python3/dist-packages/gi/overrides/GObject.py:502: Warning: ../../../gobject/gsignal.c:2735: instance '0x558ed347ba00' has no handler with id '13679'
  return func(*args, **kwargs)
/usr/lib/python3/dist-packages/gi/overrides/GObject.py:502: Warning: ../../../gobject/gsignal.c:2735: instance '0x558ed347ba00' has no handler with id '13678'
  return func(*args, **kwargs)

(rhythmbox:14426): Gtk-WARNING **: 09:51:01.314: Can't set a parent on widget which has a parent

(rhythmbox:14426): RhythmDB-CRITICAL **: 09:51:01.924: rb_refstring_unref: assertion 'g_atomic_int_get (&val->refcount) > 0' failed

(rhythmbox:14426): RhythmDB-CRITICAL **: 09:51:01.932: rb_refstring_unref: assertion 'g_atomic_int_get (&val->refcount) > 0' failed
Floating point exception (core dumped)

**Comment by ghost** [comment]
Install Coverart Again and follow the updates in page: https://github.com/fossfreedom/coverart-browser

But first: remove the plugin:

 rm -Rf ~/.local/share/rhythmbox/plugins/coverart_*

**Comment by linuxitomio** [comment]
I have just done all what you say , but seems that doesnt work yet. 

 rm -Rf ~/.local/share/rhythmbox/plugins/coverart_*
deloco@deloco:~$ sudo apt-get install git gettext python3-mako gir1.2-notify-0.7 python3-lxml python3-gi-cairo python3-cairo gstreamer1.0-plugins-ugly gstreamer1.0-plugins-good gstreamer1.0-plugins-bad rhythmbox-plugins gir1.2-webkit-3.0
[sudo] password for deloco: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package gir1.2-webkit-3.0
E: Couldn't find any package by glob 'gir1.2-webkit-3.0'
E: Couldn't find any package by regex 'gir1.2-webkit-3.0'
deloco@deloco:~$ rm -rf ~/.local/share/rhythmbox/plugins/coverart_browser
deloco@deloco:~$ git clone https://github.com/fossfreedom/coverart-browser.git
fatal: destination path 'coverart-browser' already exists and is not an empty directory.
deloco@deloco:~$ cd coverart-browser
deloco@deloco:~/coverart-browser$ ./install.sh
installing languages to /usr/share/locale/
installing en_US
en_US.po:13: warning: header field 'Language-Team' still has the initial default value
Installing the glib schema (password needed)
deloco@deloco:~/coverart-browser$ echo "deb http://ftp.de.debian.org/debian stretch main" | sudo tee -a /etc/apt/sources.list
deb http://ftp.de.debian.org/debian stretch main
deloco@deloco:~/coverart-browser$ sudo apt update
Hit:1 http://archive.canonical.com/ubuntu focal InRelease
Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [109 kB]      
Ign:3 http://ppa.launchpad.net/flozz/flozz/ubuntu focal InRelease              
Hit:4 http://se.archive.ubuntu.com/ubuntu focal InRelease                      
Ign:5 http://ftp.de.debian.org/debian stretch InRelease                        
Get:6 http://se.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]     
Get:7 http://ftp.de.debian.org/debian stretch Release [118 kB]                 
Hit:8 http://ppa.launchpad.net/gezakovacs/ppa/ubuntu focal InRelease           
Ign:9 http://ppa.launchpad.net/phablet-team/tools/ubuntu focal InRelease       
Get:10 http://security.ubuntu.com/ubuntu focal-security/main i386 Packages [172 kB]
Err:11 http://ppa.launchpad.net/flozz/flozz/ubuntu focal Release               
  404  Not Found [IP: 91.189.95.85 80]
Get:12 http://se.archive.ubuntu.com/ubuntu focal-backports InRelease [101 kB]  
Get:13 http://ftp.de.debian.org/debian stretch Release.gpg [2 410 B]           
Ign:13 http://ftp.de.debian.org/debian stretch Release.gpg                     
Err:14 http://ppa.launchpad.net/phablet-team/tools/ubuntu focal Release        
  404  Not Found [IP: 91.189.95.85 80]
Get:15 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [419 kB]
Get:16 http://se.archive.ubuntu.com/ubuntu focal-updates/main i386 Packages [401 kB]
Get:17 http://security.ubuntu.com/ubuntu focal-security/main Translation-en [94,8 kB]
Get:18 http://security.ubuntu.com/ubuntu focal-security/main amd64 DEP-11 Metadata [24,3 kB]
Get:19 http://security.ubuntu.com/ubuntu focal-security/universe i386 Packages [415 kB]
Get:20 http://se.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [732 kB]
Get:21 http://se.archive.ubuntu.com/ubuntu focal-updates/main Translation-en [183 kB]
Get:22 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [523 kB]
Get:23 http://se.archive.ubuntu.com/ubuntu focal-updates/main amd64 DEP-11 Metadata [264 kB]
Get:24 http://se.archive.ubuntu.com/ubuntu focal-updates/main amd64 c-n-f Metadata [11,5 kB]
Get:25 http://se.archive.ubuntu.com/ubuntu focal-updates/restricted i386 Packages [13,3 kB]
Get:26 http://se.archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [110 kB]
Get:27 http://se.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [19,5 kB]
Get:28 http://se.archive.ubuntu.com/ubuntu focal-updates/multiverse i386 Packages [5 860 B]
Get:29 http://se.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 DEP-11 Metadata [2 468 B]
Get:30 http://se.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [723 kB]
Get:31 http://se.archive.ubuntu.com/ubuntu focal-updates/universe i386 Packages [535 kB]
Get:32 http://se.archive.ubuntu.com/ubuntu focal-updates/universe Translation-en [146 kB]
Get:33 http://se.archive.ubuntu.com/ubuntu focal-updates/universe amd64 DEP-11 Metadata [281 kB]
Get:34 http://security.ubuntu.com/ubuntu focal-security/universe Translation-en [70,2 kB]
Get:35 http://security.ubuntu.com/ubuntu focal-security/universe amd64 DEP-11 Metadata [56,5 kB]
Get:36 http://se.archive.ubuntu.com/ubuntu focal-updates/universe DEP-11 48x48 Icons [188 kB]
Get:37 http://se.archive.ubuntu.com/ubuntu focal-updates/universe DEP-11 64x64 Icons [324 kB]
Get:38 http://security.ubuntu.com/ubuntu focal-security/universe amd64 c-n-f Metadata [9 464 B]
Get:39 http://se.archive.ubuntu.com/ubuntu focal-updates/universe amd64 c-n-f Metadata [15,1 kB]
Get:40 http://se.archive.ubuntu.com/ubuntu focal-backports/universe amd64 DEP-11 Metadata [1 768 B]
Reading package lists... Done                                                 
E: The repository 'http://ppa.launchpad.net/flozz/flozz/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: GPG error: http://ftp.de.debian.org/debian stretch Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7638D0442B90D010 NO_PUBKEY 04EE7237B7D453EC NO_PUBKEY EF0F382A1A7B6500
E: The repository 'http://ftp.de.debian.org/debian stretch Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ppa.launchpad.net/phablet-team/tools/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
deloco@deloco:~/coverart-browser$ sudo apt install gir1.2-webkit-3.0
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package gir1.2-webkit-3.0
deloco@deloco:~/coverart-browser$ sudo apt-get install git gettext python3-mako python3-requests python3-gdbm python3-pil python3-lxml
Reading package lists... Done
Building dependency tree       
Reading state information... Done
gettext is already the newest version (0.19.8.1-10build1).
git is already the newest version (1:2.25.1-1ubuntu3).
python3-mako is already the newest version (1.1.0+ds1-1ubuntu2).
python3-requests is already the newest version (2.22.0-2ubuntu1).
python3-gdbm is already the newest version (3.8.5-1~20.04.1).
python3-lxml is already the newest version (4.5.0-1ubuntu0.2).
python3-pil is already the newest version (7.0.0-4ubuntu0.1).
The following package was automatically installed and is no longer required:
  libllvm10
Use 'sudo apt autoremove' to remove it.
0 upgraded, 0 newly installed, 0 to remove and 11 not upgraded.
deloco@deloco:~/coverart-browser$

**Comment by ghost** [comment]
oh. is ubuntu??? =/ I dont know about ubuntu =( sorry!
